src/ImageSharp/Formats/Jpeg/JpegMetadata.cs

```diff
@@ -23,12 +26,46 @@ private JpegMetadata(JpegMetadata other)
         {
             this.Quality = other.Quality;
             this.ColorType = other.ColorType;
+
+            this.LumaQuantizationTable = other.LumaQuantizationTable;
+            this.ChromaQuantizationTable = other.ChromaQuantizationTable;
+            this.LumaQuality = other.LumaQuality;
+            this.ChromaQuality = other.ChromaQuality;
         }
 
+        /// <summary>
+        /// Gets or sets luminance qunatization table derived from jpeg image.
+        /// </summary>
+        /// <remarks>
+        /// Would be null if jpeg was encoded using table from ITU spec
+        /// </remarks>
+        internal Block8x8? LumaQuantizationTable { get; set; }
+
+        /// <summary>
+        /// Gets or sets chrominance qunatization table derived from jpeg image.
+        /// </summary>
+        /// <remarks>
+        /// Would be null if jpeg was encoded using table from ITU spec
+        /// </remarks>
+        internal Block8x8? ChromaQuantizationTable { get; set; }
+
+        internal double LumaQuality { get; set; }
+
+        internal double ChromaQuality { get; set; }
+
         /// <summary>
         /// Gets or sets the encoded quality.
         /// </summary>
-        public int Quality { get; set; } = 75;
+        public int Quality
+        {
+            get => (int)Math.Round(this.LumaQuality);
+            set => this.LumaQuality = value;
```

Comment:
How does luma vs chroma quality correalate in practical cases? Wouldn't it be better to drop both non-standard tables if user touches `Quality` to make the behavior symmetric? (see also my other comment)

Reply:
Recent commits made quality as average between luma and chroma but I don't think it's a very good idea (will post a more detailed answer below).